fastmri_recon/models/utils/fourier.py

```diff
@@ -255,7 +255,7 @@ def adj_op(self, inputs):
             else:
                 kspace, ktraj, shape = inputs
         if self.density_compensation:
-            kspace = tf.cast(dcomp, kspace.dtype) * kspace[..., 0]
+            kspace = tf.cast(dcomp[:, None], kspace.dtype) * kspace[..., 0]
```

Comment:
did you test this for both multicoil and singlecoil ?

I indeed was using a batch size of one even when testing..., my bad

Reply:
We test a version of this in the tf-fastmri-data. 
I have not tested for single coil.. I am not sure how extensive your testing is for this..

Reply:
Adding here that I tested it even for single coil